pages/en/lb2/Attaching-models-to-data-sources.md

```diff
@@ -81,6 +98,14 @@ you can specify the data source you want it to use from among those you've added
 [Data source generator](Data-source-generator.html) and the default `db`
 data source (that uses the [memory connector](Memory-connector.html)).
 
+```shell
```

Comment:
might wanna add something like:
to create a model, run `slc loopback:model` or `apic create --type model` (in similar format you have above)